doc/api/process.md

```diff
@@ -481,6 +482,22 @@ Would generate the output:
 4: four
 ```
 
+## process.argv0
+<!-- YAML
+added: v6.4.0
```

Comment:
It should be pretty safe to assume that this would land in `v6.4.0`, but in case it is held up or a security release intervenes, you can use `REPLACEME` in order to mark this as something that will be changed during the release itself.